service/frontend/workflowHandler.go

```diff
@@ -410,19 +410,28 @@ func (wh *WorkflowHandler) UpdateDomain(
 			return nil, err
 		}
 	} else {
+		wh.GetLogger().Info("Failover request received, checking failover permission.", tag.WorkflowDomainName(updateRequest.GetName()))
```

Comment:
Please add a tag like "OperationName" and tag with DomainUpdate for this and the other logs in this file so we can search all the domain updates in the logs.